LoRaEngine/test/LoRaWan.IntegrationTest/OTAATest.cs

```diff
@@ -87,18 +95,28 @@ public async Task Test_OTAA_Confirmed_And_Unconfirmed_Message_With_Custom_RX1_DR
 
                 await Task.Delay(Constants.DELAY_FOR_SERIAL_AFTER_SENDING_PACKET);
 
+                if (device.IsMultiGw)
```

Comment:
Does it make sense to assert here that the slow gateway is dropping the message because another gateway is processing it?
That would ensure that we are really not processing the message because we lost the race and nothing else.

Reply:
I already had a check for downstream in ABP. I added that here as well for consistency.